packages/lib-react-components/src/Loader/Loader.js

```diff
@@ -0,0 +1,69 @@
+import { string } from 'prop-types'
+import { Box } from 'grommet'
+import { useTranslation } from '../translations/i18n'
+import styled from 'styled-components'
+
+const StyledBar = styled(Box)`
+  @keyframes bounce-height {
+    0% {
+      height: 20%;
+    }
+
+    10% {
+      height: 100%;
+    }
+
+    100% {
+      height 20%;
+    }
+  }
+
+  animation: bounce-height 1s linear 0s infinite;
+  height: 20%;
+  width: 10%;
+  border-radius: 4px;
+  margin: 0 1px;
+  background: ${props => props.theme.global.colors.brand};
+
+  &.delayed-one {
+    animation: bounce-height 1s linear 200ms infinite;
+  }
+
+  &.delayed-two {
+    animation: bounce-height 1s linear 500ms infinite;
+  }
+`
+
+function Loader({
+  height = 'xxsmall',
+  loadingMessage = '',
+  width = 'xxsmall'
+}) {
+  const { t } = useTranslation()
+
+  return (
+    <Box
+      a11yTitle={loadingMessage || t('Loader.loading')}
+      role='status'
+      direction='row'
+      justify='between'
+      align='center'
```

Comment:
```suggestion
      align='center'
      alignSelf='center'
```

This should fix the alignment.

Reply:
True! I'm trying to leave positioning via styling up to the parent elements of this lightweight Loader. Good point that I missed the center alignment for the two components that use it in app-project though, so I'll correct those.